tests/system/feature_store/test_spark_engine.py

```diff
@@ -0,0 +1,78 @@
+# Marked as enterprise because of v3io mount and pipelines
+import os
+
+import pytest
+
+import mlrun.feature_store as fs
+from mlrun import store_manager
+from mlrun.datastore.sources import ParquetSource
+from tests.system.base import TestMLRunSystem
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestFeatureStoreSparkEngine(TestMLRunSystem):
+    project_name = "fs-system-spark-engine"
+    spark_service = ""
+    pq_source = "testdata.parquet"
+
+    @classmethod
+    def _init_env_from_file(cls):
+        env = cls._get_env_from_file()
+        cls.spark_service = env["DEFAULT_SPARK_SERVICE"]
+        os.environ.setdefault("V3IO_USERNAME", env["V3IO_USERNAME"])
```

Comment:
your `custom_setup` called in the base class's `setup_method` after `_setup_env` is called, so not sure why all this is needed